### DISCUSSION TITLE
Accessing Chat with Business Subscription

### DISCUSSION THREAD

**Discussion by BalagePMI** [community/community#111190] [Copilot Conversations]
### Select Topic Area

Question

### Body

I think I miss the last step...

I used the chat (beta) while I had an individual subscription, but as our company bought a business plan, I lost it.
I, as one of the owners of our organization, was able to enable the chat in Policies, but there is still no chat in my IDEA.

What is the last step I missed to access this feature?

**Comment by Akash1134** [comment]
Hi @BalagePMI ,

Please check the [eligibility criteria](https://docs.github.com/en/copilot/github-copilot-chat/using-github-copilot-chat-in-your-ide?tool=jetbrains_beta#joining-the-beta-for-github-copilot-chat-in-a-jetbrains-ide) for Copilot Chat in JetBrains IDE and ensure it is correctly [setup](https://docs.github.com/en/copilot/github-copilot-chat/using-github-copilot-chat-in-your-ide?tool=jetbrains_beta#enabling-or-disabling-github-copilot-chat-4).

Thanks!

**Reply by BalagePMI** [reply]
Thanks for the reply.

I've rechecked the criteria and if I interpreted the rules well, my organization fits to the "Owners of invoiced organizations with Account Manager" (Although it is not completely clear what an "account manager" is.)

Also, I checked if I turned on the chat globally and it is "Enabled".

![image](https://github.com/community/community/assets/109793189/8b6177a0-a954-4a8c-a7c2-8db9a27d3038)

So it all seems good, however in the IDEA I still get no access to the chat:

![image](https://github.com/community/community/assets/109793189/637409fe-9130-4b9c-bc86-309f514ef8cf)

**Comment by BalagePMI** [comment]
Ok. I still can't figure out it.
I feel my English good, but now I start to doubt I can interpret the creteria or the manual well.

My questions:

This [page](https://docs.github.com/en/copilot/github-copilot-chat/using-github-copilot-chat-in-your-ide?tool=jetbrains_beta#joining-the-beta-for-github-copilot-chat-in-a-jetbrains-ide) clearly specifies: 

> Owners of organizations with a Copilot Business subscription can decide whether to grant access to GitHub Copilot Chat in your IDE.

and 

> GitHub Copilot Chat is available to all organizations and enterprises that have an active GitHub Copilot Business license. You can enable or disable GitHub Copilot Chat for your organization or enterprise in the GitHub Copilot settings page.

Based on these my organization -- having several business subscriptions -- is eligible to use the chat.

As one of the owner of the organization, I made the global setting and enabled the chat feature in my Organization > Copilot > Policies

![image](https://github.com/community/community/assets/109793189/59fddf3f-24a2-4fd4-b56a-03a698d3f6ec)

Still In idea, I can't use the chat and gives me the screen to subscribe. 

I've even uninstalled the IDEA plugin an reinstalled it to check if there is some stayed behind data from my individual user (using the copilot before) which locks this, but didn't help.

Any idea what have I missed?

**Comment by BalagePMI** [comment]
I was able to make it work: I missed upgrading my plugin. After installing the new version, the chat has become available.
